app/services/reports/provisional_assessments.rb

```diff
@@ -0,0 +1,43 @@
+module Reports
+  class ProvisionalAssessments
+    NAME = 'provisional_assessment'.freeze
+    COLUMNS = %w[provider_name provider_type provider_claims supplier_number
+                 supplier_claims claimed authorised percent].freeze
+
+    def self.call
+      new.call
+    end
+
+    def call
+      Stats::MIData.connection.execute(query).to_a
+    end
+
+    private
+
+    def query
+      %{SELECT supplier_data.provider_name, provider_type,
```

Comment:
💭 could use [squiggly heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) for better log format and readability but either way